tests/output-eve-anomaly/test.yaml

```diff
@@ -0,0 +1,26 @@
+requires:
```

Comment:
requires should check for either: suri version 'min-version: 5.0.0' or the anomaly logger:
```
requires:
  features:
    - HAVE_LIBJANSSON
  files:
    - src/output-json-anomaly.c
```

Reply:
To clarrify: as-is the test will break on 4.1.x. It is supposed to be skipped on that version.

Reply:
Will add the a `files:` section.